contracts/MarketRegistry.sol

```diff
@@ -130,11 +137,20 @@ contract MarketRegistry is IMarketRegistry, ClearingHouseCallee, MarketRegistryS
         emit MarketMaxPriceSpreadRatioChanged(baseToken, ratio);
     }
 
-    function setFeeDiscountRatio(address trader, uint24 discountRatio) external checkRatio(discountRatio) onlyOwner {
+    function setFeeDiscountRatio(address trader, uint24 discountRatio)
+        external
+        checkRatio(discountRatio)
+        onlyOwnerAndHottubFeeManager
+    {
         _feeDiscountRatioMap[trader] = discountRatio;
         emit FeeDiscountRatioChanged(trader, discountRatio);
     }
 
+    function setHottubFeeManager(address hottubFeeManagerArg) external onlyOwner {
+        _hottubFeeManager = hottubFeeManagerArg;
```

Comment:
strongly suggest to rename `_hottubFeeManager` to `_feeManager`. curie does not need to know anything about hot tub, it's just a feature enabling another role to update fee

this repo shouldn't include anything related to hot tub. hot tub is just a dapp built on top of curie